web/src/features/AppConfig/components/AppConfig.tsx

```diff
@@ -590,31 +648,23 @@ class AppConfig extends Component {
           </SideNavWrapper>
           <div className="ConfigArea--wrapper">
             <UseIsHelmManaged>
-              {({ data = {} }) => {
-                const { isHelmManaged } = data;
-
-                const {
-                  mutate: saveConfig,
-                  isLoading: isSaving,
-                  isError: saveError,
-                } = useSaveConfig({
-                  appSlug: this.getSlug(),
-                });
+              {({ data = {} }: { data: { isHelmManaged?: boolean } }) => {
+                const { isHelmManaged: isHelmManagedFromHook } = data;
 
-                const {
-                  download,
-                  clearError: clearDownloadError,
-                  error: downloadError,
```

Comment:
this never returns an error. i think it did previously but i updated the fetch and didn't have typechecking in place to complain. added a todo in the `useDownloadValues` hook to fix this